tests/notebooks/test_moving_between_models.py

```diff
@@ -0,0 +1,25 @@
+from testbook import testbook
+from dffml import IntegrationCLITestCase
+
+
+class TestNotebook(IntegrationCLITestCase):
+    @testbook("examples/notebooks/Moving_between_models.ipynb")
+    def test_stdout(tb, self):
+        tb.execute_cell(range(0, 5))
```

Comment:
So if you notice, I have removed the call to execute the whole notebook that I previously showed you on meet. Here, we initially execute till the 5th cell, excluding the call to `cached_download` in the notebook.